samza-operator/src/test/java/org/apache/samza/operators/spec/TestOperators.java

```diff
@@ -16,22 +16,28 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.samza.operators.internal;
+package org.apache.samza.operators.spec;
 
-import org.apache.samza.operators.MessageStream;
-import org.apache.samza.operators.WindowState;
 import org.apache.samza.operators.data.Message;
+import org.apache.samza.operators.functions.FlatMapFunction;
+import org.apache.samza.operators.functions.SinkFunction;
```

Comment:
Could we ,maybe rename this to `TestOperatorSpecs` since, this appears to be testing `OperatorSpecs` for the most part?